cmd/operator/README.md

```diff
@@ -8,9 +8,9 @@ Table of Contents
 
 ## Introduction
 
-This One Operator is a Kubernetes custom controller whose goal is to serve the
+Intel Devices Plugin Operator is a Kubernetes custom controller whose goal is to serve the
```

Comment:
Should we use 'Intel Device Plugins' as in other places?

Reply:
oops, fixed.